src/models/discreteBar.js

```diff
@@ -49,6 +51,14 @@ nv.models.discreteBar = function() {
                 series.values.forEach(function(point) {
                     point.series = i;
                 });
+
+                // Changes the availableWidth to allow for a minimum bar width
+                if(minBarWidth) {
+                    var minWidth = series.values.length * minBarWidth * 1.1;
+                    if(availableWidth < minWidth) {
+                        availableWidth = minWidth;
+                    }
+                }
```

Comment:
availableWidth is used by more than the bar calculation, this should only be changing the actual bar position and width.